ci-operator/config/openshift/installer/master.json

```diff
@@ -17,13 +17,14 @@
   "test_base_image": {
     "cluster": "https://api.ci.openshift.org",
     "namespace": "openshift",
-    "name": "release",
-    "tag": "golang-1.10"
+    "name": "centos-bazel-go-oc",
```

Comment:
This changes the whole base image, I'd prefer to continue to use go-release (which is where the team needs to be).

Instead, define a new input image in `base_images`, and add a new custom image step for building your output.  Is there a docker file for that already?  Then you'd have your new cluster-launch-installer-smoke reference LOCAL_IMAGE_<name>

Reply:
If you already have a docker file in the repo you'd just define:

```
images: [
  ...
  {
     "from": "<new_base_image_name>",
     "to": "bazel-bin",
     "optional": true,
     "dockerfile_path": "images/..."
  }
```

Reply:
@stevekuznetsov said that change was coming next week-ish.. that's available now? if not, I can push the Dockerfile, have someone w/ privs oc create the is.